command/plan.go

```diff
@@ -86,12 +94,32 @@ func (c *PlanCommand) Run(args []string) int {
 		c.Ui.Error(c.Help())
 		return 1
 	}
-	file := args[0]
 
-	// Parse the job file
-	job, err := jobspec.ParseFile(file)
+	// Read the Jobfile
+	path := args[0]
+
+	var f io.Reader
+	switch path {
+	case "-":
+		if c.testStdin != nil {
+			f = c.testStdin
+		} else {
+			f = os.Stdin
+		}
```

Comment:
can you set `path = "stdin"` and then change the error msg on :113 to be `Error opening file from %v..`